src/org/usfirst/frc/team2335/robot/Robot.java

```diff
@@ -173,17 +197,47 @@ public void teleopInit()
 		{
 			autonomousCommand.cancel();
 		}
+		
+		SmartDashboard.putString("DB/String 0", Double.toString(ultraSound.getRangeBack()));
 	}
 
 	@Override
 	public void teleopPeriodic() //This function is called periodically during operator control	
 	{
 		SmartDashboard.putString("DB/String 0", Double.toString(centerX));
-    	SmartDashboard.putString("DB/String 1", Double.toString(targetWidthPx));
-    	SmartDashboard.putString("DB/String 2", Double.toString(vision.getDistance()));
+		SmartDashboard.putString("DB/String 1", Double.toString(ultraSound.getRangeBack()));
+		SmartDashboard.putString("DB/String 2", Double.toString(ultraSound.getRangeFront()));	
+		
+    	driveTrain.drive(oi.getAxis(MOVE, 1), oi.getAxis(ROTATE, 0.6));
+    	//Decreases or increases the speed of the shooter motor
+    	//If the d-pad buttons are pressed on the xbox controller
     	
-    	driveTrain.drive(oi.getAxis(MOVE, 1), oi.getAxis(ROTATE, 1));
-    	oi.printPOV();
+    	//TODO: Set range of values for the motorSpeed, so we don't kill the motor
+    	if(oi.xbox.getPOV() == 0)
+    	{
+    		shooter.speedUp();
+    	}
+    	else if(oi.xbox.getPOV() == 180)
+    	{
+    		shooter.speedDown();
+    	}
+    	
+    	//TODO: Fix this logic
+    	/*
+    	if(DriverStation.getInstance().getMatchTime() <= 20 &&
+    			DriverStation.getInstance().getMatchTime() >= 18)
+    	{
+    		oi.xbox.setRumble(RumbleType.kLeftRumble, 1);
+    		oi.xbox.setRumble(RumbleType.kRightRumble, 1);    		
+    	}
+    	else if(DriverStation.getInstance().getMatchTime() == 17)
+    	{
+    		oi.xbox.setRumble(RumbleType.kLeftRumble, 0);
+    		oi.xbox.setRumble(RumbleType.kRightRumble, 0);    
+    	}
+    	
+    	SmartDashboard.putString("DB/String 9" , Double.toString(DriverStation.getInstance().getMatchTime()));
+    	*/
```

Comment:
See a [previous comment of mine](https://github.com/SargonRobotics/Steamworks-2017/pull/5#discussion_r104558936).